src/SonarQube.Client/Models/ServerExclusions.cs

```diff
@@ -32,15 +35,21 @@ public ServerExclusions()
         {
         }
 
-        public ServerExclusions(string[] exclusions,
-            string[] globalExclusions,
-            string[] inclusions)
+        public ServerExclusions(IEnumerable<string> exclusions,
+            IEnumerable<string> globalExclusions,
+            IEnumerable<string> inclusions)
         {
-            Exclusions = exclusions ?? EmptyValues;
-            GlobalExclusions = globalExclusions ?? EmptyValues;
-            Inclusions = inclusions ?? EmptyValues;
+            Exclusions = AddPathPrefixIfNeeded(exclusions) ?? EmptyValues;
+            GlobalExclusions = AddPathPrefixIfNeeded(globalExclusions) ?? EmptyValues;
+            Inclusions = AddPathPrefixIfNeeded(inclusions) ?? EmptyValues;
         }
 
+        /// <summary>
+        /// Similarly to the other SL flavors, we will prefix everything with "**/" it's not already prefixed.
+        /// </summary>
+        private static string[] AddPathPrefixIfNeeded(IEnumerable<string> paths) =>
+            paths?.Select(path => path.StartsWith("**/") || path.StartsWith("**\\") ? path : Path.Combine("**", path)).ToArray();
```

Comment:
I think we should prefix with `**/` rather than `**\` - it will create less confusion for users if we stick to the syntax specified in the documentation.